frontend/public/index.html

```diff
@@ -14,7 +14,7 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
     />
-    <link rel="stylesheet" href="/assets/css/styles.css" />
+    <!-- <link rel="stylesheet" href="/assets/css/styles.css" /> -->
```

Comment:
Please remove it entirely.

Reply:
Agreed

Reply:
Link has been removed and new PR request has been created.